evaluate2.sh

```diff
@@ -149,9 +150,13 @@ for fork in "$@"; do
 
     # Linux platform
     # prepend this with numactl --physcpubind=0-7 for running it only with 8 cores
-    numactl --physcpubind=0-7 hyperfine $HYPERFINE_OPTS "./calculate_average_$fork.sh 2>&1"
-  else
-    hyperfine $HYPERFINE_OPTS "./calculate_average_$fork.sh 2>&1"
+    numactl --physcpubind=0-7 hyperfine $HYPERFINE_OPTS "timeout -v $RUN_TIME_LIMIT ./calculate_average_$fork.sh 2>&1"
+  else # MacOS
```

Comment:
Ah, very nice. So `evaluate2.sh` runs on MacOS too?

Reply:
Yes, I've been testing on both Fedora Linux and my MacBook M2 Air

Reply:
Do we need to check whether `gtimeout` command is available?

Reply:
I have `timeout` rather than `gtimeout` on MacOS. Should we just use that?

Reply:
LOL, nevermind, wrong console ;)

It is checked already, so LGTM. Will merge. Will only add log message for this case.